mediacapture-streams/MediaStreamTrack-idl.https.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<title>MediaStreamTrack IDL tests</title>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#media-stream-track-interface-definition">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/resources/WebIDLParser.js></script>
+<script src=/resources/idlharness.js></script>
+<script>
```

Comment:
When I did an IDL test, I found that if I put the IDL inside <textarea>, I could cut-and-paste it without having to do end-of-line escaping.
But if you're going for separate IDL files eventually, it's not worth it.